### ISSUE TITLE
add to mkda chi squared description

### ISSUE THREAD

**Issue by nicoalee** [neurostuff/neurostore#632]
We want the mkda chi squared description to include: "compare to reference dataset". I believe this is a change to be done to the metadata file representing the various nimare supported algorithms

**Comment by adelavega** [comment]
Is this mean to say this is not hardcoded on the frontend? Isn't this already existing on the frontend?

**Comment by nicoalee** [comment]
Yeah, the file with algorithm/corrector names/descriptions is generated and not hardcoded on the frontend.
